Tasks/4 - Input Field/ScriptWorkdays.js

```diff
@@ -0,0 +1,24 @@
+function calculateWorkingDays() {
```

Comment:
The filename `ScriptWorkdays.js` is written in what is called **PascalCase** style. PascalCase is commonly used for React applications, but it is not typical for vanilla JavaScript code. For this current situation, it is better to use **kebab-case**, e.g., `script-workdays.js`

But it is better to remove the `Script` prefix and just name the file `workdays.js`

Reply:
done